bots/Bots.md

```diff
@@ -31,6 +31,10 @@
   - [sg\_single\_rule\_delete](#sg_single_rule_delete)
   - [tag\_ec2\_resource](#tag_ec2_resource)
   - [vpc\_turn\_on\_flow\_logs](#vpc_turn_on_flow_logs)
+  - [s3\_block\_public\_access](#s3_block_public_access)
```

Comment:
You don't work correctly with git, you have here 2 bots but you wanted to add only one to this pr.
Feel free to contact if you miss something, it is very simple

Reply:
done